freqtrade/exchange/kraken.py

```diff
@@ -48,8 +55,8 @@ def get_balances(self) -> dict:
 
             orders = self._api.fetch_open_orders()
             order_list = [(x["symbol"].split("/")[0 if x["side"] == "sell" else 1],
-                           x["remaining"] if x["side"] == "sell" else x["remaining"] * x["price"],
-                           # Don't remove the below comment, this can be important for debugging
+                          x["remaining"] if x["side"] == "sell" else x["remaining"] * x["price"],
+                          # Don't remove the below comment, this can be important for debugging
                            # x["side"], x["amount"],
```

Comment:
if you must remove these 2 spaces - then please remove the 3rd one too so the 2 comments align...